### PR TITLE
Update min LN machine account balance

### PR DESCRIPTION
**Pull request by jordanschalm**
To match https://github.com/onflow/flow/pull/732

### CODE REVIEW

**Review by joshuahannan — APPROVED** [review]
👍

File: module/epochs/machine_account.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func init() {
 	if err != nil {
 		panic(fmt.Errorf("could not convert soft min balance for LN: %w", err))
 	}
-	HardMinBalanceLN, err = cadence.NewUFix64("0.001")
+	HardMinBalanceLN, err = cadence.NewUFix64("0.002")
```
Comment (tarakby) [inline module/epochs/machine_account.go:44]: Not related to this change, but shouldn't we increase `SoftMinBalanceSN` to cover the upper bound of DKG broadcasts?

**Review by tarakby — APPROVED** [review]

File: module/epochs/machine_account.go (hunk 1 above)
Comment (jordanschalm) [reply module/epochs/machine_account.go:44]: It should already cover worst-case upper bound for DKG, see https://www.notion.so/dapperlabs/Machine-Account-f3c293593ea442a39614fcebf705a132#1e3b627ce81c4d4195d99c67a1958ebb

Comment (tarakby) [reply module/epochs/machine_account.go:44]: 💡 Nice, yes that's what I meant!

**Comment by jordanschalm** [comment]
bors merge

**Pull request merged**